book_source/03_topical_pages/03_pecan_xml.Rmd

```diff
@@ -775,3 +776,46 @@ This information is currently used by the following R functions:
 
 Coming soon...
 
+### Remote data module {#xml-remote_process}
+
+This section describes the tags required for configuring `remote_process`.
+
+```xml
+  <remotedata>
+  <out_get_data>...</out_get_data>
+  <source>...</source>
+  <collection>...</collection>
+  <scale>...</scale>
+  <projection>...</projection>
+  <qc>...</qc>
+  <algorithm>...</algorithm>
+  <credfile>...</credfile>
+  <out_process_data>...</out_process_data>
+  <overwrite>...</overwrite>
+  </remotedata>
+```
+
+* `raw_mimetype`: (required) MIME type of the raw file
```

Comment:
:eyes:

Reply:
yup the docs need to be updated to explain the registry file logic and tags